### DISCUSSION TITLE
how to uninstall and go back to "standard integration"

### DISCUSSION THREAD

**Discussion by Dobbiiey** [humbertogontijo/homeassistant-roborock#392] [Q&A]
Hello & first sorry for the stupid question.
I tried out the custom integration. For installing I followed the description from this git-repo.
Now I would like to go back to standard integration.
I removed the integration in HACS (I don't "see" it there anymore).
if I go to Settings->Devices & Services I can only find & select the custom integration and don't see the standard/official.
Thanks for helping
Tobi

**Comment by Lash-L** [comment]
You need to restart after deleting it from hacs and removing the device from the integrations page

**Reply by Dobbiiey** [reply]
@Lash-L , thanks for helping.
Strange situation: I had still a folder "/config/custom_components/roborock" on the drive & installing "standard version" didn't work until I deleted this folder...
Now it works like it should.
